src/cryptoadvance/specter/liquid/wallet.py

```diff
@@ -502,6 +503,53 @@ def adjust_fee(self, psbt, fee_rate):
             * (tx_full_size / psbt["tx"]["vsize"])
         )
 
+    def addresses_info(self, is_change, asset_labels=[]):
```

Comment:
Passing asset_labels to all functions is not necessary, see comment below - `<asset-label>` component can fetch the label by itself.